Detections/ZoomLogs/SupiciousLinkSharing.yaml

```diff
@@ -0,0 +1,32 @@
+id: 1218175f-c534-421c-8070-5dcaabf28067
+name: Suspicious link sharing pattern
+description: |
+  'Alerts in links that have been shared across multiple Zoom chat channels by the same user in a short space if time. 
+  Adust the threshold figure to change the number of channels a message needs to be posted in before an alert is raised.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - ZoomLogs
+queryFrequency: 24h
+queryPeriod: 24h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - Persistence
+relevantTechniques:
+  - T1192
+query: |
+
+  let threshold = 3; 
+  let lookback = 24h; 
+  ZoomLogs 
+  | where TimeGenerated> ago(lookback) 
+  | where Event == "chat_message.sent" 
+  | extend Channel = tostring(parse_json(ChatEvents).Channel)  
+  | extend Message = tostring(parse_json(ChatEvents).Message) 
+  | where Message matches regex "http(s?):\\/\\/" 
+  | summarize dcount(Channel) by Message, User 
```

Comment:
Needs to bring thru time - StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), should we also bring thru Channel and UserId?

Reply:
makeset(Channel) would be needed to bring thru Channel